packages/tools/accessibility/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@dev/accessibility",
+    "name": "@tools/accessibility",
     "description": "Render accessibility tree for 3D content in DOM, thus the 3D content is compatible with screen readers and tab selection",
     "private": false,

```

Comment:
should probably stay private, as this will not be released to npm.

Reply:
resurfacing this - the PR will not be merged before this is set to true, otherwise npm will try releasing this during the publish process.